nornir/core/helpers/jinja_helper.py

```diff
@@ -11,9 +11,12 @@ def render_from_file(path, template, jinja_filters=None, **kwargs):
     return template.render(**kwargs)
 
 
-def render_from_string(template, jinja_filters=None, **kwargs):
+def render_from_string(template, path=None, jinja_filters=None, **kwargs):
     jinja_filters = jinja_filters or {}
-    env = Environment(undefined=StrictUndefined, trim_blocks=True)
+    if path:
```

Comment:
this should be `if path is not None`

Reply:
Hi,
Nope, it is if path, because if path is none call the function without loader, if it isn't none call the function with Loader

Reply:
you need to account for things that are not `None` but might be `falsey`. For instance, if the user passes `path=""`, that's why `if path` should be `if path is not None`

Reply:
Nice! didn't thought about it! thanks!